src/Directory.Build.props

```diff
@@ -0,0 +1,31 @@
+<Project>
+
+  <PropertyGroup>
+    <FrameworkPathOverride Condition="'$(TargetFramework)' == 'net461' and '$(MONO_ROOT)' != ''">$(MONO_ROOT)/lib/mono/4.6.1-api/</FrameworkPathOverride>
+    <FrameworkPathOverride Condition="'$(TargetFramework)' == 'net47' and '$(MONO_ROOT)' != ''">$(MONO_ROOT)/lib/mono/4.7-api/</FrameworkPathOverride>
+    <LangVersion>latest</LangVersion>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
+    <NoWarn>$(NoWarn);1591</NoWarn>
+    <DebugType>portable</DebugType>
+    <DebugSymbols>True</DebugSymbols>
+    <RepositoryName>RepoName</RepositoryName>
+    <PackageLicenseUrl>https://raw.githubusercontent.com/Faithlife/$(RepositoryName)/master/LICENSE</PackageLicenseUrl>
```

Comment:
`RepoName` was a literal token that got replaced [here](https://github.com/Faithlife/CSharpTemplate/wiki#modify-the-code).

I'm not opposed to changing it to something more "variable-y" but we'd have to change the Cake script, README, and other places. Might be best to leave it as `RepoName` here (unless you're always planning on injecting it as a variable to have fewer merge conflicts down the road?).

Reply:
`RepoName` will be replaced in the previous line the same way. No other changes are needed. I pulled it into a property because the previous method caused merge conflicts, e.g. when we changed the release notes URL.

Reply:
I totally missed that. :+1: